### PR TITLE
Adding in Forms

### PR DESCRIPTION
**Pull request by tig4647**
Added for List of forms in shipment. Added in below scanForm

### CODE REVIEW

File: EasyPost/Shipment.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ public class Shipment : IResource {
         public PostageLabel postage_label { get; set; }
         public List<Rate> rates { get; set; }
         public ScanForm scan_form { get; set; }
+         public List<Forms> forms { get; set; }
```
Comment (att14) [inline EasyPost/Shipment.cs]: You've got an extra space on this line.

**Comment by att14** [comment]
Thanks for this. These two pull request can be merged into one.

**Comment by tig4647** [comment]
Sorry about the spacing. I will figure out GitHub yet!

**Comment by tig4647** [comment]
To be honest, I am not sure how to merge the 2 changes. I plan to take time learning GitHub this weekend on pluralsight. I am sorry if my contributions arent helpful. I am a single developer and just recently started using GitHub to contribute.

**Comment by att14** [comment]
No problem. I can handle this if you would like. But, you can't really do this on GitHub. Just use git.

```
git checkout patch-2
git merge patch-1
git commit
git push origin HEAD
```

Then close patch-1 on GitHub.

**Comment by DarkChowder** [comment]
I have created a new pull request to replace this one, as it has not been corrected. This pull request does not add forms as a reference to the projects, and would cause shipments.cs to not compile.

**Pull request closed without merging**